roles/phantomjs/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Accept the MS TrueType fonts installer EULA
+  command: echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
```

Comment:
Will this command work if it happens before the `ttf-mscorefonts-installer` package is installed below?